src/icp_frontend/pages/Members.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState, useEffect } from 'react'
+import {
+  Button,
+  Table,
+  Alert,
+  Container,
+  Form,
+  Row,
+  Col,
+  Spinner,
+} from 'react-bootstrap'
+import { chapter_2 } from '../../declarations/chapter_2'
+import 'bootstrap/dist/css/bootstrap.min.css'
+import { Link } from 'react-router-dom'
+
+function Members() {
+  const [members, setMembers] = useState([])
+  const [isFetched, setIsFetched] = useState(false)
+
+  const handleMembers = async () => {
+    try {
+      const membersData = await chapter_2.getAllMembers()
+      setMembers(membersData)
+      setIsFetched(true)
+      console.log('Members:', membersData)
+    } catch (error) {
+      console.error('Error fetching members:', error)
+      setIsFetched(true)
```

Comment:
if your doing `setIsFetched(true)` in both  `try` and `catch`  move it to the [`finally` block](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch#syntax)